scalafix-tests/input/src/main/scala/test/DisableRegex.scala

```diff
@@ -29,12 +29,9 @@ object DisableRegex {
   import scala.collection.mutable   // ok
 
   @SuppressWarnings(Array("Disable.ListBuffer"))
-  @SuppressWarnings(Array("Disable.<init>"))
   val buffer = new ListBuffer[Int]()
```

Comment:
What happens if you have a more complex qualifier in a select?

```
a.b(1).c
```
and `a` and `c` are banned? I would expect there to be two lint errors
```
a.b(1).c
^

a.b(1).c
       ^
```

Reply:
![Colombo: one more thing](http://www.ttmc.co.uk/media/209316/dchat-1443540044-13-blog-metv_blog_columbo_justonemorethingcompressed.jpg)

Reply:
I opened https://github.com/scalacenter/scalafix/issues/679 and assigned the ticket to you @vovapolu . I will go ahead and merge to unblock a release, but please be sure to follow up on that ticket as I think the current change results in surprising behavior.